Dockerfile

```diff
@@ -39,6 +40,18 @@ ENV PRE_CREATE_DB **None**
 ENV SSL_SUPPORT **False**
 ENV SSL_CERT **None**
 
+
+# Add ContainerPilot
+RUN curl -Lo /tmp/cb.tar.gz https://github.com/joyent/containerpilot/releases/download/2.1.0/containerpilot-2.1.0.tar.gz \
+&& tar -xz -f /tmp/cb.tar.gz \
+&& mv ./containerpilot /bin/
+COPY containerpilot.json /etc/containerpilot.json
+COPY start.sh /start.sh
+RUN chmod +x /start.sh
+
+ENV CONSUL=consul:8500
```

Comment:
can we remove the CONSUL env variable by default? Else I'll have an issue using it in other contexts